Ginger/Ginger/ApplicationModelsLib/APIModels/APIModelPage.xaml.cs

```diff
@@ -40,7 +40,7 @@ public partial class APIModelPage : Page
         ApplicationAPIModel mApplicationAPIModel;
         ModelParamsPage page;
         private bool saveWasDone = false;
-        public APIModelPage(ApplicationAPIModel applicationAPIModelBase)
+        public APIModelPage(ApplicationAPIModel applicationAPIModelBase, bool IsReadOnlyRequest = false)
```

Comment:
instead IsReadOnlyRequest bool, use ViewMode enum like we do in pages like ActionEditPage, ActivityEditPage, etc..